rust/rsc/src/rsc/main.rs

```diff
@@ -199,9 +201,11 @@ async fn connect_to_database(
     create_remote_db(config).await
 }
 
-fn launch_eviction(conn: Arc<DatabaseConnection>, tick_interval: u64, deadline: i64) {
+fn launch_eviction(conn: Arc<DatabaseConnection>, tick_interval: i64, deadline: i64) {
```

Comment:
They are now both u64

The specific/silly API reasons that I needed u64 for one and i64 for the other and figured it made since to just hide the conversion inside of the function so that consumers of `launch_eviction` didn't have to think about it but I think I managed to sidestep those headaches.